src/implementations/twig/components/timeline/__snapshots__/timeline.test.js.snap

```diff
@@ -483,29 +483,13 @@ exports[`Timeline renders correctly with extra attributes 1`] = `
           >
             EU budget for the future
           </a>
-          : European Commission's legislative proposals for spending programmes
-        </div>
-      </div>
-    </li>
-    <li
-      class="ecl-timeline__item ecl-timeline__item--collapsed"
-      id="17"
-    >
-      <div
-        class="ecl-timeline__tooltip"
-      >
-        <div
-          class="ecl-timeline__tooltip-arrow"
-        />
-        <div
-          class="ecl-timeline__label"
-        >
-          28-29 June 2018
-        </div>
-        <div
-          class="ecl-timeline__content"
-        >
-          Commission contribution to the European Council: 
+          : European Commission's legislative proposals for spending programmes',
+    },
```

Comment:
there is some issue with the data here

Reply:
Oh yeah, there was, but this wouldn't have worked in the previous implementation, i adapted it so that it will work also when the "/example" is part of a longer string and not just a prop value per se... ;)